Composer/packages/client/src/components/CreationFlow/CreateBot.tsx

```diff
@@ -264,22 +278,43 @@ export function CreateBot(props: CreateBotProps) {
 
   return (
     <Fragment>
+      {/* <TemplateFeedModal
+        closeDialog={() => setIsTemplateFeedModalVisible(false)}
+        feedUrl={templateFeedUrl}
+        hidden={!isTemplateFeedModalVisible}
+        onUpdateFeed={(feed: string) => {
+          setTemplateFeedUrl(feed);
+        }}
+      /> */}
       <DialogWrapper isOpen={isOpen} {...dialogWrapperProps} dialogType={DialogTypes.CreateFlow} onDismiss={onDismiss}>
-        <Pivot
-          defaultSelectedKey={csharpFeedKey}
-          onLinkClick={(item) => {
-            if (item) {
-              setSelectedProgLang(item);
-            }
-          }}
-        >
-          <PivotItem data-testid="dotnetFeed" headerText="C#" itemKey={csharpFeedKey}></PivotItem>
-          <PivotItem
-            data-testid="nodeFeed"
-            headerText={formatMessage('Node (Preview)')}
-            itemKey={nodeFeedKey}
-          ></PivotItem>
-        </Pivot>
+        <Stack horizontal styles={tabAndFeedBtnStyles}>
```

Comment:
why was this change needed to wrap everything in a stack? seems like this should be reverted since we no longer need the edit feeds button?

Reply:
This is now a deep link that navigates you to the templae feed form in the settings page so the stack implementation is still needed